src/test/java/io/axoniq/demo/university/faculty/write/createcourseplain/CreateCourseAxonFixtureTest.java

```diff
@@ -0,0 +1,46 @@
+package io.axoniq.demo.university.faculty.write.createcourseplain;
+
+import io.axoniq.demo.university.faculty.FacultyAxonTestFixture;
+import io.axoniq.demo.university.faculty.events.CourseCreated;
+import io.axoniq.demo.university.shared.ids.CourseId;
+import org.axonframework.test.fixture.AxonTestFixture;
+import org.junit.jupiter.api.*;
+
+class CreateCourseAxonFixtureTest {
```

Comment:
Ah, only now do I note you duplicated the tests. One with fixtures one with your custom fixture solution. Was that intended?

Reply:
Makes total sense to me to keep both, Mateusz. Fair choice! I'll resolve this comment right away.